### ISSUE TITLE
Introduce a Desktop section

### ISSUE THREAD

**Issue by felipeborges** [developer-portal/content#94]
Fedora is a good and popular development platform for Gtk+ and Qt development. Writing GUI applications in these toolkits involve knowledge of tools, languages, and guidelines.

I suggest adding a Desktop section with articles such "Writing a Hello World application in gjs" (gjs is gnome javascript, the language we focus on for newcomers). I am willing to contribute with this introductory article for Gtk + gjs.

**Comment by phracek** [comment]
well It's a nice idea. On the workshop OpenAlt2015 I have heard about Gnome-Builder.

Are you able to provide a some notes or even the whole structure?

**Comment by felipeborges** [comment]
I would suggest having a Desktop section, and under it a Gtk+ and a Qt subsections.

For instance, In the Gtk+ section I would have a main article describing the toolkit and the interesting utils for Gtk development, such as: GNOME Builder, Devhelp, gtk3-demo. After that I would link to specific articles for writing a "Hello World" application in a language of your choice: gjs, pygi (python + gobject introspection), vala, C/C++, etc.. In the end of each article we could link to the upstream documentation which is more complete and less platform specific. Someone from the Qt side of the force could write something similar.

If you give me an ACK for that, I could start working in a pull request for it.

**Comment by strzibny** [comment]
That would be really nice!

**Comment by phracek** [comment]
What languages do you prefer to use? C/C++ or Python? Or both?

**Comment by phracek** [comment]
This is awesome. Go ahead. We need more GUI:)

**Comment by phracek** [comment]
I have created a DevAssistant GUI a long time ago with pygi too.

**Comment by asamalik** [comment]
@felipeborges I have marked this as 'in progress' because you have started to work on it. Is that ok?

**Comment by felipeborges** [comment]
@asamalik sure. More articles to come!

**Comment by vishnuavenu** [comment]
Can't We use Node environment in gnome apps development ?

**Comment by strzibny** [comment]
I guess prefered lang is Vala at this moment but many others are supported. I think it is fine to introduce all runtimes/envs that are primarily supported by Gnome.

**Comment by mrennekamp** [comment]
Hi, I suppose this should be multi-page like the Web Applications docs are. Should the intro page cover the basic process of GUI programming, such as game development (pygame/godot) and common everyday software (web browsers, LibreOffice)?

**Comment by strzibny** [comment]
Yeah... I agree that the focus should be on building on Fedora (not necessarily just for Fedora, though).
